packages/ui/src/components/va-tabs/VaTabs.vue

```diff
@@ -472,6 +481,10 @@ export default defineComponent({
     .va-tabs__tabs {
       position: absolute;
       height: 100%;
+
```

Comment:
Might be easier to use `v-if="showPagination && !$props.hidePagination"` on buttons in component template.

Reply:
new pr: https://github.com/epicmaxco/vuestic-ui/pull/2605
 ```css
 .va-tabs__tabs {
      position: absolute;
}
```
`position:absolute` need to be remove, let width grow by content